Mono.Cecil/AssemblyWriter.cs

```diff
@@ -3090,7 +3090,7 @@ static string GetXmlSecurityDeclaration (SecurityDeclaration declaration)
 
 		void WriteTypeReference (TypeReference type)
 		{
-			WriteUTF8String (TypeParser.ToParseable (type));
+			WriteUTF8String (TypeParser.ToParseable (type, false));
```

Comment:
Note: also had to use the same trick in `NativePdbWriter` when importing `using static` which refers to a type, PDB encode it with full qualification even for mscorlib.